addons/rating/models/rating_mixin.py

```diff
@@ -21,18 +22,22 @@ class RatingParentMixin(models.AbstractModel):
         compute="_compute_rating_percentage_satisfaction", compute_sudo=True,
         store=False, help="Percentage of happy ratings")
     rating_count = fields.Integer(string='# Ratings', compute="_compute_rating_percentage_satisfaction", compute_sudo=True)
+    rating_avg = fields.Float('Average Rating', compute='_compute_rating_percentage_satisfaction', compute_sudo=True, search='_search_rating_avg')
+    rating_avg_percentage = fields.Float('Average Rating (%)', compute='_compute_rating_percentage_satisfaction', compute_sudo=True)
```

Comment:
Would add a groups on it, unless you want it to be readable to other people ? As anyway all ratings fields are already group user.